test/Microsoft.ML.Tests/Microsoft.ML.Tests.csproj

```diff
@@ -2,6 +2,9 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp2.0</TargetFramework>
   </PropertyGroup>
+  <ItemGroup>
+    <Compile Remove="Scenarios\Api\AspirationalExamples.cs" />
```

Comment:
>AspirationalExamples [](start = 35, length = 20)

why?

Reply:
Because right now it doesn't compile, and may never compile, since it uses my own imaginary version of Pigsty, which may differ from the real one, when it appears.

---
In reply to: [210986324](https://github.com/dotnet/machinelearning/pull/688#discussion_r210986324) [](ancestors = 210986324)

Reply:
Why we even add this file? What's the point?

---
In reply to: [211030708](https://github.com/dotnet/machinelearning/pull/688#discussion_r211030708) [](ancestors = 211030708,210986324)

Reply:
Umm, as an aspirational example? It's a way to document what we want to reach at the end. Another way would be to put it into Markdown somewhere, but I think I like this way somewhat better.

---
In reply to: [211688654](https://github.com/dotnet/machinelearning/pull/688#discussion_r211688654) [](ancestors = 211688654,211030708,210986324)